From 4759b0d46316a94035a2c2c93f5eaf265a6d5090 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <aisheng.dong@freescale.com>
Date: Wed, 28 Oct 2015 16:24:59 +0800
Subject: [PATCH 0639/5242] MLK-11781-1 dts: imx6ul-evk: add non-removable
 property for TF card

commit  fdce4440722b9a657ac1cfd16676d5ed426cc3a9 from
https://source.codeaurora.org/external/imx/linux-imx.git

The TF card slot on uSDHC2 does not support CD function,
so add non-removable property.

Signed-off-by: Dong Aisheng <aisheng.dong@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ul-14x14-evk.dtsi |    1 +
 arch/arm/boot/dts/imx6ul-9x9-evk.dts    |    1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi b/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi
index 3f8b0e9..58003a7 100644
--- a/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi
+++ b/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi
@@ -397,6 +397,7 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_usdhc2>;
 	no-1-8-v;
+	non-removable;
 	keep-power-in-suspend;
 	wakeup-source;
 	status = "okay";
diff --git a/arch/arm/boot/dts/imx6ul-9x9-evk.dts b/arch/arm/boot/dts/imx6ul-9x9-evk.dts
index ec84ec7..5aa932c 100644
--- a/arch/arm/boot/dts/imx6ul-9x9-evk.dts
+++ b/arch/arm/boot/dts/imx6ul-9x9-evk.dts
@@ -760,6 +760,7 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_usdhc2>;
 	no-1-8-v;
+	non-removable;
 	keep-power-in-suspend;
 	enable-sdio-wakeup;
 	status = "okay";
-- 
1.7.9.5

